guide/designing-products/user-research.md

```diff
@@ -113,7 +113,7 @@ The Bitcoin protocol works with addresses and has otherwise no sense of identity
    layout = "float-left"
 %}
 
-While we cannot know who is directly responsible for the activity, we can analyze the overall network activity. No other currency in history has had a complete and accurate database of all transactions. What can we learn from this data that allows us to improve the system for users?
+While we often can’t know who is responsible for specific single transactions, we can still analyze overall network activity. No other currency in history has offered such a complete and accurate database of transactions. Here’s how we can use this data to improve bitcoin for users:
```

Comment:
```suggestion
While we often can’t know who is responsible for specific single transactions, we can still analyze overall network activity. No other currency in history has offered such a complete and accurate ledger of transactions. Here’s how we can use this data to improve Bitcoin for users:
```

Reply:
There isn't examples listed after the last statement which implies there is some.